### ISSUE TITLE
Add Interp specifier to CesiumSunSky to change date and time in Sequencer

### ISSUE THREAD

**Issue by argallegos** [CesiumGS/cesium-unreal#760]
A user mentioned that certain properties are not accessible in the Sequencer, such as SunSky date and time information. This means that things like sunrises, etc can't be easily animated using Sequencer.

These properties can be exposed to Sequencer by adding the `Interp` specifier when declaring the UProperty.

**Comment by mathieuvannereux** [comment]
Hi, can you explain how to do this please ? Where do you "declaring the UProperty" ?

**Comment by kring** [comment]
There's an unmerged PR that adds it, so you can take a look at that:
https://github.com/CesiumGS/cesium-unreal/pull/761

There are also comments in there about how you may be able to make use of these changes.

**Comment by mathieuvannereux** [comment]
Hi, I already read the topic #761 with the Repeater Event Track, but it seems to work only by "Adding interp specifier to relevant properties". After searching explanations on wen, and if I understand well, I have to modify the code source of the Cesium Sun & Sky with C++ in Virtual Studio ? But I don't know how to do that, the tutorials I found with Virtual Studio are completely wooly for me !

**Comment by kring** [comment]
Right, you currently need to modify the plugin. If you're not comfortable doing that, then unfortunately changing the time with the sequencer is not possible at this time.

The only other possibility is to create a new custom property with the Interp attribute on one of your own objects, and code it so that setting your property sets the corresponding CesiumSunSky property. It _might_ even possible to do this from Blueprints, but I haven't tried.

**Comment by mathieuvannereux** [comment]
Thank you, I'm not afraid of trying to code but I would just like to know where to start. I don't even know where to access to these "properties", whether in BP or C++ ..
